docs/zkapps/o1js/smart-contracts.md

```diff
@@ -229,51 +212,34 @@ class HelloWorld extends SmartContract {
 }
 ```
 
-Here, after obtaining the current state `x` and asserting that it equals the on-chain value, we make another assertion:
+Here, after obtaining the current state `x` and asserting that it equals the on-chain value, make another assertion:
 
 ```ts
 x.add(1).assertEquals(xPlus1);
 ```
 
-If the assertion fails, o1js will throw an error and not submit the transaction.
-On the other hand, if it succeeds, it becomes part of the proof that is verified on-chain.
-
-Because of this, our new version of `increment()` is _guaranteed_ to behave like the previous version: It can only ever
-update the state `x` to `x + 1`.
+If the assertion fails, o1js throws an error and does not submit the transaction.
+If it succeeds, it becomes part of the proof that is verified on-chain.
```

Comment:
What are the two "it"s referring to in this sentence?

Reply:
"the assertion"